### ISSUE TITLE
A question about POG paper

### ISSUE THREAD

**Issue by WxxShirley** [wenyuer/P0G#1]
Hi,
I have just read your paper in KDD 19. It is really a nice work! 
I am confused about the training process. In the POG framework, are the parameters in transition layers(both in Per and Gen ) are sharable with FOM models' transition layer parameters?
Or they are all learned during the training process with the proposed loss function?

Thanks!

**Comment by wenyuer** [comment]
Hi Xixi,


Thank you for your interest.
The parameters in POG's Gen network are initialized as the pre-trained FOM, so as the transition layers.&nbsp;
And also they can be learned during the training process, if your dataset is large enough.&nbsp;




Best Regards




Wen Chen (陈雯)


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Xixi Wu"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月8日(星期天) 下午3:40
收件人:&nbsp;"wenyuer/POG"<POG@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[wenyuer/POG] A question about POG paper (#1)




Hi,
 I have just read your paper in KDD 19. It is really a nice work!
 I am confused about the training process. In the POG framework, are the parameters in transition layers(both in Per and Gen ) are sharable with FOM models' transition layer parameters?
 Or they are all learned during the training process with the proposed loss function?
 
Thanks!

**Comment by WxxShirley** [comment]
I got it. 
Thanks!

**Comment by hyyuan123** [comment]
Hi,Is there a public code for pog papers?
